.github/workflows/check.yml

```diff
@@ -94,7 +94,7 @@ jobs:
           key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock', 'patches/*.patch') }}
       - run: yarn || yarn --network-concurrency 1
       - run: yarn check-licenses
-  depcheck:
+  knip-depcheck:
```

Comment:
maybe instead of replacing depcheck immediately we could keep depcheck and run knip depcheck as a non-required CI step for a while to see if it's working?

Reply:
seems possible that we'll notice some categories we don't care about, like unused enum members or something, and we'll want to update the knip config. would be nice if we weren't blocked with our other work every time we find something like that